mdf_connect_server/automate/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019, IRIS-HEP
```

Comment:
Is this right? MDF is CHiMaD/NIST funded, and our repos are licensed under Apache 2.0.